File: engine/engine-model/src/main/scala/net/server/channel/packet/ring/HandleWishListPacket.scala
package net.server.channel.packet.ring

class HandleWishListPacket(private var _mode: Byte, private var _amount: Int, private var _items: Array[String]) extends BaseRingPacket(_mode = _mode) {
  def amount: Int = _amount

  def items: Array[String] = _items
}

File: engine/engine-model/src/main/java/net/server/channel/packet/ring/HandleWishListPacket.java
package net.server.channel.packet.ring;

public class HandleWishListPacket extends BaseRingPacket {
   private final Integer amount;

   private final String[] items;

   public HandleWishListPacket(Byte mode, Integer amount, String[] items) {
      super(mode);
      this.amount = amount;
      this.items = items;
   }

   public Integer amount() {
      return amount;
   }

   public String[] items() {
      return items;
   }
}
